### ISSUE TITLE
Riders Republic (2290180)

### ISSUE THREAD

**Issue by Celsiusss** [ValveSoftware/Proton#7036]
# Compatibility Report
- Name of the game with compatibility issues: Riders Republic
- Steam AppID of the game: 2290180

## System Information
- GPU: RX 6950 XT
- Video driver version: Mesa 23.1.6
- Kernel version: 6.4.11
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Celsiusss/3bd85870d59bcfef3ccb8a8e134651ed
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-2290180.log](https://github.com/ValveSoftware/Proton/files/12407822/steam-2290180.log)

## Symptoms 
Game crashes on startup
![image](https://github.com/ValveSoftware/Proton/assets/3050747/3398fddd-e69a-4a44-96d5-2a63607361d3)

## Reproduction

Start the game, wait for Ubisoft launcher, see Riders Republic splash screen, then a window saying the game crashed pops up.

**Comment by Blisto91** [comment]
@Celsiusss Thank you for the report. This should now have been worked around in the latest Proton Experimental.

**Comment by Celsiusss** [comment]
> @Celsiusss Thank you for the report. This should now have been worked around in the latest Proton Experimental.

Great work! Tested it and it works :)

**Comment by trinukso** [comment]
> @Celsiusss Thank you for the report. This should now have been worked around in the latest Proton Experimental.

It is true that the game now starts and you can start playing. But after five minutes the game inevitably ends up crashing.

**Comment by Blisto91** [comment]
Can you add a proton log from a launch where it crashes? 🙂 
Add `PROTON_LOG=1 %command%` to the game's launch options and attach the log which  appears in your users home folder.

**Comment by trinukso** [comment]
> Can you add a proton log from a launch where it crashes? 🙂 Add `PROTON_LOG=1 %command%` to the game's launch options and attach the log which appears in your users home folder.

Yes, of course. Thank you very much. However, I have the game purchased from the Epic store and I run it with "Heroic Launcher" using "Proton Experimental". I hope that won't be a drawback. Apologies.

[steam-heroic-RidersRepublic.log](https://github.com/ValveSoftware/Proton/files/12776670/steam-heroic-RidersRepublic.log)

**Comment by trinukso** [comment]
> Can you add a proton log from a launch where it crashes? 🙂 Add `PROTON_LOG=1 %command%` to the game's launch options and attach the log which appears in your users home folder.

Yes, of course. Thank you very much. However, I have the game purchased from the Epic store and I run it with "Heroic" using Proton Experimental. I hope this is not an inconvenience. Apologies.

[steam-heroic-RidersRepublic.log](https://github.com/ValveSoftware/Proton/files/12776676/steam-heroic-RidersRepublic.log)
